bin/sail

```diff
@@ -269,7 +269,7 @@ if [ $# -gt 0 ]; then
         if [ "$EXEC" == "yes" ]; then
             docker-compose exec \
                 mysql \
-                bash -c 'MYSQL_PWD=${MYSQL_PASSWORD} mysql -u ${MYSQL_USER} ${MYSQL_DATABASE}'
```

Comment:
Single quotes are preferred here because it then evaluates the command inside the container shell, instead of your local shell.

Reply:
Oh my, I actually didn't want to change the quotes in here, this was a mistake on my side